WPThemeReview/Sniffs/Templates/RequiredFunctionSniff.php

```diff
@@ -0,0 +1,231 @@
+<?php
+/**
+ * WPThemeReview Coding Standard.
+ *
+ * @package WPTRT\WPThemeReview
+ * @link    https://github.com/WPTRT/WPThemeReview
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WPThemeReview\Sniffs\Templates;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * Ensures that body_class() is called if theme adds a <body> tag.
+ *
+ * @link  https://make.wordpress.org/themes/handbook/review/required/#templates
+ *
+ * @since 0.1.0
+ */
+class RequiredFunctionSniff implements Sniff {
+
+	/**
+	 * Sniff Settings
+	 *
+	 * @var array
+	 */
+	public $tagsConfig = array(
```

Comment:
`public` sniff properties can be changed from a custom ruleset. Was this done intentionally? If not you can change visibility to `protected` 🙂

Reply:
Well,  I don't know the stance on that for this repo, but it was intentional mostly for my own use case, which probably isn't very common.  I figured if I had a need to tweak something, then someone somewhere might as well.  I did name this as a "config" to indicate it was able to be configured and is an open point, so others can implement as needed.  I generally like to leave configurations open as much as possible because you never know what someone might want to modify :smile:, but I completely understand if that's not a standard approach here.